clusters/cijioagents2.yaml

```diff
@@ -0,0 +1,24 @@
+---
+helmDefaults:
+  atomic: true
+  force: false
+  timeout: 300
+  wait: true
+repositories:
+  # https://github.com/DataDog/helm-charts/
+  - name: datadog
+    url: https://helm.datadoghq.com
+  # https://github.com/jenkins-infra/helm-charts/
+  - name: jenkins-infra
+    url: https://jenkins-infra.github.io/helm-charts
+releases:
+  - name: docker-registry-secrets
+    # This helm chart doesn't create any resources within the namespace specified below.
+    # Specifying a namespace is required by the "needs" feature of helmfile (to allow referencing to this release from others)
+    namespace: default
+    chart: jenkins-infra/docker-registry-secrets
+    version: 0.1.0
+    values:
+      - "../config/docker-registry-secrets.yaml"
+    secrets:
+      - "../secrets/config/docker-registry-secrets/secrets.yaml"
```

Comment:
As per https://github.com/jenkins-infra/helpdesk/issues/4460, this release is not needed anymore. Let's simplify as much as we can.

It means that you have to open this PR to install `datadog` instead of `docker-registry-secrets`